src/components/NavBar/SearchBar.css.ts

```diff
@@ -103,7 +103,7 @@ export const suggestionImage = sprinkles({
 export const suggestionPrimaryContainer = style([
   sprinkles({
     alignItems: 'flex-start',
-    width: 'full',
```

Comment:
We only want to add sprinkles if they're used frequently. Since this is a one-off, can you instead after the closing bracket of `sprinkles` create a new obj with the 90% width style. It will look like this:
```
export const suggestionPrimaryContainer = style([
  sprinkles({
    alignItems: 'flex-start',
  }),
  {
    width: '90%',
  }
])  
```

Reply:
thank you! makes sense, fixed.